src/PaneDetails/PaneDetails.css

```diff
@@ -0,0 +1,6 @@
+/* Pane Details */
+.closeButton {
+  font-size: 30px;
```

Comment:
We have `--font-size-xx-large`, it's not exactly 30px (it's about 31.5px) , but maybe it's better than hardcoded value? What do you think?

Reply:
I tried to set using variables but didn't work. I ended up using pixel and I think its better than hardcoded values, thanks man.